source/fe/fe_enriched.cc

```diff
@@ -727,7 +727,8 @@ FE_Enriched<dim, spacedim>::multiply_by_enrichment(
             continue;
 
           Assert(enrichments[base_no - 1][m](cell) != nullptr,
-                 ExcMessage("The pointer to the enrichment function is NULL"));
+                 ExcMessage(
+                   "The pointer to the enrichment function is nullptr"));
```

Comment:
Could we change this to something clearer like "The pointer to the enrichment function is not set"?

Reply:
Fixed.